TSD 194 prim sec toggle

gkalvatcheva opened the pull request:
- responsive adjustments of new toggle
- footer fix for small screen sizes

gkalvatcheva committed:
- style of top dashboard toggle
- variable addition
- mixin addition for border radius when we want it on left or right only
- slight structure change of the toggle

gkalvatcheva committed:
- styles for language nav and new toggle in small screen sizes

gkalvatcheva committed:
Merge branch 'edudash-2.0' into TSD-194-prim-sec-toggle

gkalvatcheva committed:
- capitalization of translation word for schools
- adjustments in responsive to accommodate for the new toggle

gkalvatcheva committed:
Merge branch 'edudash-2.0' into TSD-194-prim-sec-toggle

gkalvatcheva committed:
- dashboard toggle adjustment
- responsive footer adjustment

gkalvatcheva committed:
- reused some variables

gkalvatcheva committed:
- adjustment for main nav visibility on ipad

gkalvatcheva committed:
- removed text wrap for translated toggle
-removed background in active lang nav

esoliani commented on app/i18n/locale_en.json:
```diff
@@ -13,7 +13,7 @@
     "tab.national-view": "National View",
     "tab.school-view": "Schools",
-    "tab.school": "schools",
+    "tab.school": "Schools",
```
this should be done using style capitalize

uniphil replied:
I think it's ok -- we can `style: lowercase` if we need it lowered, which has less edge cases for CSS to mess up than `capitalize`.

esoliani commented on app/styles/responsive.css:
```diff
@@ -1,138 +1,114 @@
-@media only screen and (max-width: 1280px) {
```
why are we version this file? and why are there errors on it?

uniphil replied:
good catch. removed.

gkalvatcheva committed:
Merge branch 'edudash-2.0' into TSD-194-prim-sec-toggle

uniphil committed:
change array form module def to normal function

uniphil committed:
Clean up header/footer directives

I thought they could be replaced by `<div ng-include="template">`s, but that doesn't seem to work.

uniphil committed:
Keep primary/secondary toggle in sync with binding

uniphil committed:
Merge remote-tracking branch 'origin/edudash-2.0' into TSD-194-prim-sec-toggle

uniphil committed:
Clean up unneeded css-related files

uniphil committed:
Rename partials to start with _

uniphil committed:
Merge remote-tracking branch 'origin/edudash-2.0' into TSD-194-prim-sec-toggle
